server/apps/archive/takes_package_service.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8; -*-
```

Comment:
Yes, we can move other modules also if there are any related to package into one folder module.